lib/specs/canary.js

```diff
@@ -35,6 +35,15 @@ let rules = {
         details: oneLineTrim`Please change <code>"ghost-api"</code> in your <code>package.json</code> to higher version. E.g. <code>{"engines": {"ghost-api": "v4"}}</code>.<br>
         If <code>"ghost-api"</code> property is left at "v2", it will stop working with next major version upgrade and default to v5.<br>
         Check the <a href="${docsBaseUrl}packagejson/" target=_blank><code>package.json</code> documentation</a> for further information.`
+    },
+    'GS001-DEPR-LABS-MEMBERS': {
+        level: 'warning',
+        rule: 'The <code>{{@labs.members}}</code> helper should not be used and removed from the theme.',
+        details: oneLineTrim`Please remove <code>{{@labs.members}}</code> from the theme.<br>
+        The <code>{{@labs.members}}</code> helper will default to "true" value in Ghost v4 and will be removed from Ghost v5, thus resolving to "false" value.
```

Comment:
```suggestion
        The <code>{{@labs.members}}</code> helper will always return <code>true</code> in Ghost v4 and will be removed from Ghost v5, at which point it will return <code>null</code> and evaluate to <code>false</code>.
```